src/main/java/frc/robot/RobotContainer.java

```diff
@@ -78,26 +106,22 @@ public class RobotContainer {
 
     SlewRateLimiter limitX = new SlewRateLimiter(6);
     SlewRateLimiter limitY = new SlewRateLimiter(6);
+    SlewRateLimiter limitI = new SlewRateLimiter(6);
```

Comment:
Can you put a comment on what these SlewRateLimiter's are for? `limitI` and `limitS` are not very informative.

Reply:
Also is there a reason you need them in the first place? Generally SlewRateLimiter is to limit acceleration, but I would think you want shooter and intake to speed up as fast as possible

Reply:
Sure I'll add comments. We originally didn't have limiters for the intake and shooter but then running the shooter from 0 to 50+ percent would kill comms

Reply:
You should be able to set the rampRate on the motor controller directly.